### ISSUE TITLE
Enter and Space select a Tab, but Navigation Does Not Happen

### ISSUE THREAD

**Issue by frastlin** [vaadin/vaadin-tabs#165]
Hello,
In both mobile and desktop view, if you select a tab and hit enter with the keyboard without firing the onClick event, the tab shows as selected, but the page does not change.
This is extremely undesirable behavior.
Thanks,

**Comment by web-padawan** [comment]
The `vaadin-tab` component has code that performs `click()` on the anchor Enter / Space keys:

https://github.com/vaadin/vaadin-tabs/blob/05ba06a4511fc387dce9e82360a4dd2d01bde9d3/src/vaadin-tab.html#L68-L73

To me it's unclear why this doesn't work. Could you please provide a code example?

**Comment by frastlin** [comment]
Hello,
I think our team was able to figure it out. I'm not exactly sure what they did, but it works now.
